raven/src/main/java/com/getsentry/raven/util/Util.java

```diff
@@ -109,4 +109,22 @@ public static Double parseDouble(String value, Double defaultValue) {
         }
         return Double.parseDouble(value);
     }
+
+    /**
+     * Trims a String, ensuring that the maximum length isn't reached.
+     *
+     * @param string string to trim
+     * @param maxMessageLength maximum length of the string
+     * @return trimmed string
+     */
+    public static String trimString(String string, int maxMessageLength) {
```

Comment:
It might be worth considering ellipisizing this? I'm not sure what behavior is consistent among the other SDKs…?